client/src/components/core/ms-filter/MsFilterPopover.vue

```diff
@@ -0,0 +1,96 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-list class="filter-container">
+    {{ $t('UsersPage.filter.status') }}
+    <div>
+      <user-status-tag :revoked="false" />
+      <ion-checkbox
+        aria-label=""
+        class="checkbox"
+        :checked="props.filters.statusActive"
```

Comment:
You can replace it all with

```vue
<ion-checkbox
  aria-label=""
  class="checkbox"
  v-model="filters.statusActive"
/>
```
The v-model will automatically bind the value of the checkbox to our variable, which means that the value will be updated without having to dismiss the popover.